### ISSUE TITLE
Missing icon: gnome-chess

### ISSUE THREAD

**Issue by ubuntujaggers** [snwh/suru-icon-theme#102]
I didn't realise that there was a whole suite of Gnome games, other than the three that ship with Ubuntu.

Maybe something like:

![image](https://user-images.githubusercontent.com/38893390/46752576-80fcd400-ccb5-11e8-9a4d-ccaa4f149a23.png)

![image](https://user-images.githubusercontent.com/38893390/46739085-a549b800-cc97-11e8-9973-9b3fdf2b8741.png)

?

I've tried lighter grey squares as per the second (2D) screenshot at https://en.wikipedia.org/wiki/GNOME_Chess - using the darkest grey on the Suru palette.

Alternatively: same basic design as you have in Moka?

**Comment by snwh** [comment]
🤔 I think this could be better executed. If the pawn is to be the pictogram (i.e. carrying the main icon metaphor) it should be more prominent in the icon–see how in your screenshot the music note, the A and the folder are as large as it is on the canvas in the other icons.

**Comment by ubuntujaggers** [comment]
Fair point.  If the piece is to be more prominent I wonder if it should be something more distinguished, like the king?

![chess king](https://user-images.githubusercontent.com/38893390/46827189-e9b87f00-cd8f-11e8-9f79-d788dd8f3b5d.png)

![chess king](https://user-images.githubusercontent.com/38893390/46827283-25534900-cd90-11e8-8161-9d69725cfe88.png)

**Comment by ubuntujaggers** [comment]
Work in progress of a bigger chess piece:

![image](https://user-images.githubusercontent.com/38893390/46837388-1a5ae180-cdad-11e8-8c7f-2fbd7c094e07.png)

![image](https://user-images.githubusercontent.com/38893390/46836847-0d3cf300-cdab-11e8-8763-4b75795e5e08.png)

**Comment by ubuntujaggers** [comment]
Alternatively: is a chessboard pattern by itself, iconic enough to stand for chess?  That would allow for more contrast between the squares.  The difficulty I'm having is making a recognisable chess piece at the two smallest sizes.

![image](https://user-images.githubusercontent.com/38893390/46975129-b4789d80-d0bd-11e8-9295-e31114c9e474.png)

![image](https://user-images.githubusercontent.com/38893390/46975153-c3f7e680-d0bd-11e8-9a7a-3023a14cec6c.png)

**Comment by ubuntujaggers** [comment]
@snwh 

What do you think of this one:

![image](https://user-images.githubusercontent.com/38893390/47086569-cb300900-d211-11e8-9ffe-04c8b4a3f692.png)

![image](https://user-images.githubusercontent.com/38893390/47086592-d97e2500-d211-11e8-9074-f5c4e665f6bc.png)

?

**EDIT:** I'm tempted to remove the chess piece from the smallest size and let the four squares stand for chess at 16x16.

**EDIT 2:** With lighter squares...

![chesssssssssss 7 svg](https://user-images.githubusercontent.com/38893390/47111695-17983a80-d24c-11e8-99f1-6fe1a5779e37.png)

![image](https://user-images.githubusercontent.com/38893390/47097930-43a2c400-d22a-11e8-954c-ce1b3a28840d.png)
